Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,23 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
+    system "./configure"
+    system "make"
+    bin.install "main/ch_track"
+    bin.install "main/ch_wave"
+    bin.install "main/pda"
+    bin.install "main/pitchmark"
+    bin.install "main/sig2fv"
+    bin.install "main/sigfilter"
+    bin.install "main/spectgen"
+    bin.install "main/wagon"
+  end
+
+  test do
+    assert_equal "Error reading ESPS file -", `ch_track 2>&1`.strip
```

Comment:
Yeh, `make test` runs unit tests but we want to test that the utilities were compiled/installed correctly by running something afterwards.